ddtrace/ext/errors.py

```diff
@@ -4,17 +4,28 @@
 
 import traceback
 
+from ddtrace.constants import ERROR_MSG
+from ddtrace.constants import ERROR_STACK
+from ddtrace.constants import ERROR_TYPE
+from ddtrace.utils.deprecation import deprecated
+from ddtrace.utils.deprecation import deprecation
 
-ERROR_MSG = "error.msg"  # a string representing the error message
-ERROR_TYPE = "error.type"  # a string representing the type of the error
-ERROR_STACK = "error.stack"  # a human readable version of the stack. beta.
 
-# shorthand for -----^
+__all__ = [ERROR_MSG, ERROR_TYPE, ERROR_STACK]
+
+deprecation(
+    name="ddtrace.ext.errors",
+    message="Use `ddtrace.constants` module instead",
```

Comment:
Can we mention here that `MSG`, `TYPE` and `STACK` are deprecated and to use `ERROR_MSG`, `ERROR_TYPE` and `ERROR_STACK` instead?